internal/openshift/user_channel_map.go

```diff
@@ -1,35 +1,40 @@
 package openshift
 
 import (
-	"github.com/fabric8-services/fabric8-jenkins-idler/internal/model"
 	"sync"
+
+	"github.com/fabric8-services/fabric8-jenkins-idler/internal/model"
 )
 
+// UserChannelMap is a type-safe and concurrent map storing channels for model.User keyed against the user namespace
 type UserChannelMap struct {
 	sync.RWMutex
 	internal map[string]chan model.User
 }
 
-// A type-safe and concurrent map storing channels for model.User keyed against the user namespace.
+// NewUserChannelMap creates a new instance of UserChannelMap
```

Comment:
@kishansagathiya We lost some information here, didn't we?

Reply:
https://github.com/fabric8-services/fabric8-jenkins-idler/pull/142/files/534eda6b77bfb7eaae118b0c15948f6f2e247f0a#diff-86cb10d5a84e282eda5bce50248f298cR9